### ISSUE TITLE
Pixel 5 Missing Cellular Settings in Secondary Users

### ISSUE THREAD

**Issue by VSfz3kkDwtjm4LAD2R24** [GrapheneOS/os_issue_tracker#472]
Hi, I'm using RQ1A.210205.004.2021.02.26.16 on Pixel 5.

Unlike my Pixel 4 XL on GrapheneOS, it seems like on Pixel 5 secondary users don't have detailed cellular settings but just a single tile control in the notification center. Whether intentional or not, it might be an improvement for security, but it's somehow inconvenient especially considering secondary users cannot even initiate cellular hotspot.

I would really appreciate it if the cellular settings could be added back, or I would also be more than happy to be educated with the reasons behind the missing functionality. Thanks a lot!

**Comment by thestinger** [comment]
Are both using the same carrier?

**Comment by thestinger** [comment]
I don't think this is a difference between the devices but rather a carrier difference. If you have a separate SIM card for each phone, try switching them around.

**Comment by VSfz3kkDwtjm4LAD2R24** [comment]
After I switched to Pixel 5, I flashed my Pixel 4 XL back to stock. Let me flash GrapheneOS again tonight and try it. I'll get it back to you. Thanks!

**Comment by thestinger** [comment]
For example, some carriers don't have APN configuration available. You *must* use their standard APN. It's possible for the OS to ignore that it's supposed to work this way, but at the moment, we follow the standard carrier configuration profiles.

In all likelihood this is a difference between stock OS and GrapheneOS though. Stock OS may have chosen to preserve legacy behavior in some cases for backwards compatibility.

**Comment by VSfz3kkDwtjm4LAD2R24** [comment]
It's weird because, first there was no problem when I still had Pixel 4 XL using GrapheneOS with the same carrier and number. Second, the issue only applies to non-admin users on my Pixel 5. Under admin user, all cellular settings are present and the hotspot tile can also be enabled in the notification center.

**Comment by VSfz3kkDwtjm4LAD2R24** [comment]
*I also make sure I have enabled call and sms for the non-admin user

**Comment by VSfz3kkDwtjm4LAD2R24** [comment]
Never mind. I just tested on stock Android and it behaves exactly the same as GrapheneOS, where detailed cellular settings aren't available to non-admin users. Either something got changed or there is something wrong with my memory. Most likely the latter. I apologize for any inconvenience and confusion. I'll go ahead and close this issue.

Again, thanks a lot for the explanation!

**Comment by thestinger** [comment]
You can enable tethering on the owner user and then it will keep working when you switch users. The owner user is always running in the background.

**Comment by thestinger** [comment]
Unlike secondary users, which have their sessions ended automatically if you go over the active limit and there's also the end session action in GrapheneOS.

Owner is in control of most global settings. Some things are *oddly* profile-specific in a way that doesn't make complete sense but over time it's being sorted out better.

**Comment by thestinger** [comment]
Don't really see how cellular settings could have been per-profile but I don't recall ever looking at that.

**Comment by VSfz3kkDwtjm4LAD2R24** [comment]
Understood. Yes, I've noticed that settings like location are user specific for some reason while some like Wi-Fi are global.

For cellular settings, today I wanted to connect a device to my phone's hotspot and then I suddenly found out the option wasn't there like it used to be. I might have recalled when I was in admin user or something.

Actually, is it okay that I opened another issue if I encountered a problem or had a question here, or do developers prefer those being posted on the Matrix channel first? I'm just worried if I were causing any trouble by opening possibly misleading issues like this. Until only recently I started trying to get into software development. I still know so little, but I do want to at least help as much as I can. I want to make sure I'm not contributing negativity to your workflow.

Thanks a lot!
